index.js

```diff
@@ -23,10 +23,16 @@ try {
   // Fetch inputs
   const name = core.getInput('name');
   const environment = core.getInput('environment');
+  const domain = core.getInput('domain');
+  const output = branchName(github.context).toLowerCase().replace(/[^a-z0-9-]/g, "").substr(0, 63)
 
   const deploy = deployName(name, environment, github.context)
   console.log(`name: ${deploy}`);
   core.setOutput('name', deploy);
+
+  const url = `${deploy}.${domain}`
+  core.setOutput('url', url);
```

Comment:
Not really sure why `url` is added? Was this used?

Reply:
I checked the `.github/workflows/review.yml ` in Siren, url is used under the deploy job.

Reply:
Um I'm really confused here now - why this shows add add when this literally is already in main: https://github.com/itsmycargo/action-deploy-environment/blob/main/index.js#L32

Perhaps rebase ...?

Reply:
I just rebased and everything is up to date 😕

Reply:
With conflicts :)

Reply:
https://github.com/itsmycargo/action-deploy-environment/pull/1/commits/df9e86ab3609d61ae4c077efe92827f3e1d08321